assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/TotalUserCount.js

```diff
@@ -20,13 +20,112 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME as CORE_SITE } from '../../../../../googlesitekit/datastore/site/constants';
+import { STORE_NAME as CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+import { STORE_NAME, DATE_RANGE_OFFSET } from '../../../datastore/constants';
+import { numberFormat, readableLargeNumber } from '../../../../../util';
+import { getAvailableDateRanges } from '../../../../../util/date-range';
+import { isZeroReport } from '../../../util';
+import ChangeArrow from '../../../../../components/ChangeArrow';
+const { useSelect } = Data;
 
-// eslint-disable-next-line no-unused-vars
 export default function TotalUserCount( { dimensionName, dimensionValue } ) {
-	return null;
+	const url = useSelect( ( select ) => select( CORE_SITE ).getCurrentEntityURL() );
+	const dateRange = useSelect( ( select ) => select( CORE_USER ).getDateRange() );
+	const dateRangeDates = useSelect( ( select ) => select( CORE_USER ).getDateRangeDates( {
+		compare: true,
+		offsetDays: DATE_RANGE_OFFSET,
+	} ) );
+
+	const args = {
+		...dateRangeDates,
+		metrics: [ { expression: 'ga:users' } ],
+		orderby: {
+			fieldName: 'ga:users',
+			sortOrder: 'DESCENDING',
+		},
+		limit: 1,
+	};
+
+	if ( url ) {
+		args.url = url;
+	}
+
+	if ( dimensionName && dimensionValue ) {
+		args.dimensions = [ dimensionName ];
+		args.dimensionFilters = {
+			[ dimensionName ]: dimensionValue,
+		};
+	}
+
+	const loaded = useSelect( ( select ) => select( STORE_NAME ).hasFinishedResolution( 'getReport', [ args ] ) );
+	const error = useSelect( ( select ) => select( STORE_NAME ).getErrorForSelector( 'getReport', [ args ] ) );
+	const report = useSelect( ( select ) => select( STORE_NAME ).getReport( args ) );
+
+	if ( ! loaded || error || isZeroReport( report ) ) {
+		return null;
+	}
```

Comment:
It's okay to have the less granular handling and just return `null` in all of those cases, but in that case we need to make sure that the `UserCountGraph` component has the more granular handling that we usually do:
* Return `PreviewBlock` for loading state
* Return `ReportError` on error
* Return `ReportZero` for zero data

cc @tofumatt

Reply:
Yes, we need to return `null` here to make sure it doesn't take extra space when the report is being fetched, a response is empty, or is an error. The `UserCountGraph` component will take all spaces and display the appropriate error/zero/preview component.